### PR TITLE
FastIR for Linux + Linux Evidence Collection category

### PR DESCRIPTION
**Pull request by hslatman**
Also some small typos fixed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,8 +48,8 @@ A curated list of tools and resources for security incident response, aimed to h
 * [Microsoft User Mode Process Dumper](http://www.microsoft.com/en-us/download/details.aspx?id=4060) - The User Mode Process Dumper (userdump) dumps any running Win32 processes memory image on the fly
 
 ### Timeline tools
-* [Plaso](https://github.com/log2timeline/plaso) -  a Python-based backend engine for the tool log2timeline
-* [Timesketch](https://github.com/google/timesketch) -open source tool for collaborative forensic timeline analysis
+* [Plaso](https://github.com/log2timeline/plaso) -  a Python-based backend engine for the tool log2timeline.
+* [Timesketch](https://github.com/google/timesketch) - open source tool for collaborative forensic timeline analysis.
```
Comment (meirwah) [inline README.md]: I'm going with the notion of without dot in end of line, can you remove those?

**Comment by hslatman** [comment]
Done, reversed them for the entries that I changed, but there are more dots throughout the file. There are also longer entries that contain multiple lines having dots.

**Comment by meirwah** [comment]
try to update from master branch

**Comment by hslatman** [comment]
Hmm, weird, thought I was up-to-date already. Should work now.

**Comment by meirwah** [comment]
:+1:

**Pull request merged**